### PR TITLE
Update PHP-cs-fixer settings and fix errors

### PR DESCRIPTION
**Pull request by thePanz**
This is a first step towards the PSR-12
No additional changes have been made apart from running the `make fix-phpcs` command

### CODE REVIEW

File: src/Aggregation/BucketScript.php [hunk 1]
```diff
@@ -71,7 +71,8 @@ public function toArray(): array
     {
         if (!$this->hasParam('buckets_path')) {
             throw new InvalidException('Buckets path is required');
-        } elseif (!$this->hasParam('script')) {
```
Comment (ruflin) [inline src/Aggregation/BucketScript.php]: Was this converted automatically? I think both options will work but they are not 100% identical I would argue.

Comment (thePanz) [reply src/Aggregation/BucketScript.php]: yes, all the changes were done by the cs-fixer

Comment (thePanz) [reply src/Aggregation/BucketScript.php]: well, it is identical. as the if branch is throwing an exception :)

Comment (ruflin) [reply src/Aggregation/BucketScript.php]: damn, true :-D `thePanz 1 : 0 ruflin`

**Pull request merged**